reference/managed-delivery/resource-status/index.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: single
+title:  "Resource Status"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+*If you're not sure what a Managed Resource is, check out our [overview of Managed Delivery](/reference/managed-delivery) first.*
+
+## Overview
+Managed Resources will always have a **status** which describes the current state of the resource from Spinnaker's perspective. A resource's status can help you quickly answer questions like:
+  - Is Spinnaker taking automatic action on this resource right now?
+  - Does the actual configuration of this resource match the declarative configuration I gave to Spinnaker?
+  - Is something wrong with this resource?
+
+## How to find a resource's status
+
+### UI
+On the Infrastructure views (Clusters, Load Balancers, Firewalls), you'll see a color-coded "M" logo attached to any resources that are managed by Spinnaker. Hovering over the logo reveals additional context about the current status and what it means.
+
+Example on a cluster:
+{%
+  include
+  figure
+  image_path="./cluster-resource-status-ui.png"
+%}
+
+
+### API
+If you're [setup to use the API](/reference/managed-delivery/getting-started/#env-setup) and have the [ID of a resource](/reference/managed-delivery/getting-started/#find-a-resources-id) you can use it to call the `/managed/resources/<resourceId>/status` endpoint.
+
+Example request:
+```bash
+curl -X GET --header "X-SPINNAKER-USER: ${SPINNAKER_USER}" "${SPIN_URL}/<resourceId>/status"
+```
+Example response:
+```json
+{ "status": "HAPPY" }
+```
+
+## Status Reference
+
+### Happy
+{%
+  include
+  figure
+  image_path="./resource-status-happy.png"
+  caption="Example of a resource with the Happy status"
+%}
+
+When a resource has a Happy status, all is well and the actual state of the resource matches its declarative configuration.
+
+If Spinnaker detects a drift from the declarative configuration in the future, the resource will change status to [Diff](/reference/managed-delivery/resource-status/#diff). Once Spinnaker starts taking automatic action to correct the drift, the status will change to [Actuating](/reference/managed-delivery/resource-status/#actuating).
+
+While Spinnaker is managing a resource, manual changes made via the UI or the API will be automatically reversed ("stomped") in favor of the declarative configuration.
+
+### Created
+{%
+  include
+  figure
+  image_path="./resource-status-created.png"
+    caption="Example of a resource with the Created status"
+%}
+
+When a resource has a Created status, it means Spinnaker was just told to start management and hasn't taken any action yet.
```

Comment:
Instead of “Spinnaker was just told to start management”, perhaps “Spinnaker has just received the resource’s declarative configuration” (or something along those lines)? I always immediately think “who told it to do anything?” when that verb is used.

Reply:
love it, very agree.